.github/workflows/docker.yaml

```diff
@@ -64,6 +64,22 @@ jobs:
         kubectl set image deployment.apps/celery celery=$DOCKER_SLUG:${GITHUB_SHA::7} -n=notification-canada-ca --kubeconfig=$HOME/.kube/config
         kubectl set image deployment.apps/celery-beat celery-beat=$DOCKER_SLUG:${GITHUB_SHA::7} -n=notification-canada-ca --kubeconfig=$HOME/.kube/config
         kubectl set image deployment.apps/celery-sms celery-sms=$DOCKER_SLUG:${GITHUB_SHA::7} -n=notification-canada-ca --kubeconfig=$HOME/.kube/config
+
+    - name: Obtain a GitHub App Installation Access Token
+      id: githubAppAuth
+      run: |
+        TOKEN="$(npx obtain-github-app-installation-access-token ci ${{ secrets.TOKEN }})"
+        echo "::add-mask::$TOKEN"
+        echo "::set-output name=token::$TOKEN"
+
+    - name: Test the obtained token
+      run: |
+        curl -X POST -H 'Content-Type: application/json' \
+          -d '{"context":"test","state":"success"}' \
+          "https://api.github.com/repos/$GITHUB_REPOSITORY/statuses/$GITHUB_SHA?access_token=$GITHUB_TOKEN"
+      env:
+        GITHUB_TOKEN: ${{ steps.githubAppAuth.outputs.token }}
```

Comment:
Did you test the commands replicated by these steps locally?

Reply:
Probably a good idea